app/locales/fr.json

```diff
@@ -0,0 +1,99 @@
+{
+  "label": {
+    "about_title": "À propos",
+    "authorities_add_button_label": "Ajouter une source de confiance",
+    "authorities_add_url": "Ajouter une autorité par URL",
+    "authorities_desc": "Choisissez une autorité sanitaire de confiance dans votre région pour obtenir les données d’exposition. Vous pouvez sélectionner un nom dans la liste globale, ou entrer l’adresse web fournie par une autorité qui a mis en place Safe Paths.",
+    "authorities_input_placeholder": "Collez votre URL ici",
+    "authorities_no_sources": "Aucune source ajoutée",
+    "authorities_removal_alert_cancel": "Annuler",
+    "authorities_removal_alert_desc": "Êtes-vous sûr·e de vouloir retirer cette autorité de sources de données?",
+    "authorities_removal_alert_proceed": "Continuer",
+    "authorities_removal_alert_title": "Supprimer une autorité",
+    "authorities_title": "Sources de confiance",
+    "choose_provider_subtitle": "Pour vous tenir informé·e des risques d’exposition, vous devez souscrire à une autorité sanitaire.",
+    "choose_provider_title": "Choisissez votre autorité sanitaire",
+    "commitment": "Engagement",
+    "commitment_para": "Safe Paths enregistre et vérifie de façon sécurisée vos interactions avec autrui en utilisant vos données de position. Vos données ne quitteront JAMAIS votre appareil sans votre consentement.",
```

Comment:
"donnée de position" doesn't sound French native. 

I would use "coordonnées" which is used often.

Reply:
Position is almost definitely fine in native French. The sentence structure does make it sound weird, so I changed that.

Plus it’s also coherent with the google export that is suggested
![image](https://user-images.githubusercontent.com/6126377/79206801-29a66980-7e40-11ea-92bd-480936608ec4.png)